lib/flop_phoenix.ex

```diff
@@ -666,9 +668,12 @@ defmodule Flop.Phoenix do
       of a column this way.
       """
 
-    attr :attrs, :list,
+    attr :attrs, :any,
       doc: """
-      Any additional attributes to pass to the `<td>`.
+      Additional attributes to pass to the `<td>` element. May be provided as a
+      static keyword list, or as a 1-arity function to dynamically generate the
+      list using row data. Note that these attributes will override any
+      conflicting `tbody_td_attrs` that are set at the table level.
```

Comment:
> Note that these attributes will override any conflicting `tbody_td_attrs` that are set at the table level.

Just a note that this particular behavior was already true, but wasn't laid out in the documentation. I assumed, probably naively, that the attrs `class` value would augment the classes that are set in the table config.